package.json

```diff
@@ -113,7 +113,7 @@
     "webpack": "3.4.1",
     "webpack-cli": "3.2.1",
     "webpack-dev-middleware": "1.11.0",
-    "webpack-dev-server": "3.1.14",
+    "webpack-dev-server": "2.9.1",
```

Comment:
?

Reply:
seems to be an issue even with latest package (webpack/webpack-dev-server#1355 (comment)) revert to 2.9.1 allow to compile without issues